content/en/docs/guides/metadata.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Metadata
+description: >-
+  Explains what metadata is, how it is transmitted, and what it is used for.
+---
+
+### Overview
+
+Metadata is a side channel that allows clients and servers to provide
+information to each other that is associated with the data messages.
+
+gRPC metadata is a key-value pair of data that is sent with a gRPC request or
+response. It is used to provide additional information about the call, such as
+authentication credentials, tracing information, or custom headers.
+
+gRPC metadata is implemented using HTTP/2 headers. The keys and values are ASCII
+strings, but the values can also be binary data. The keys are case insensitive
+and must not start with the prefix grpc-, which is reserved for gRPC itself.
+
+gRPC metadata can be sent and received by both the client and the server. It is
+sent in the initial headers of the RPC call, and can also be sent in subsequent
+trailers.
+
+gRPC metadata is useful for a variety of purposes, such as:
+
+* **Authentication**: gRPC metadata can be used to send authentication credentials to
+the server. This can be used to implement different authentication schemes, such
+as OAuth2 or JWT.
```

Comment:
It might be fair to mention "using the standard HTTP Authorization header" here, to make it more concrete.

Reply:
done